fe/fe-core/src/main/java/com/starrocks/sql/analyzer/CatalogAnalyzer.java

```diff
@@ -125,8 +140,15 @@ public Void visitAlterCatalogStatement(AlterCatalogStmt statement, ConnectContex
                         (ModifyTablePropertiesClause) statement.getAlterClause();
                 Map<String, String> properties = modifyTablePropertiesClause.getProperties();
 
+                String catalogType = GlobalStateMgr.getCurrentState().getCatalogMgr().getCatalogType(statement.getCatalogName());
                 for (Map.Entry<String, String> property : properties.entrySet()) {
-                    if (!property.getKey().equals("ranger.plugin.hive.service.name")) {
+                    String confName = property.getKey();
+
+                    if (("hive".equals(catalogType) || "hudi".equals(catalogType))) {
```

Comment:
why need to check catalog type? other catalog type can also alter catalog properties

Reply:
ok